Gemfile

```diff
@@ -2,7 +2,7 @@ source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 gemspec
 
 group :extra do
-  gem 'rugged', '>= 0.24.0', '< 0.24.6', :platforms => :ruby
+  gem 'rugged', '<= 0.28.5', :platforms => :ruby
```

Comment:
@adrienthebo just noticing we're pinning to an ancient rugged version. It first glance, it doesn't look like it can compile on Ubuntu 20.04 anymore

I'm also noticing that we're installing the pre-compiled rugged binary for the Alpine-based container, which is pulling in 0.27.5-r0 - i.e see https://travis-ci.org/github/puppetlabs/r10k/jobs/680807669#L524

You can also verify directly against latest published image like

> docker run -it --rm --entrypoint /usr/bin/ruby puppet/r10k -e "require 'rugged'; puts Rugged::VERSION"
```0.27.5```

And to make things a little more fun, I see that there's a 1.0 release out there as of April 3rd - https://rubygems.org/gems/rugged/versions

Reply:
As a test, that bump in the gemspec to 0.28.5 compiles just fine on Ubuntu 20.04. Will try another bump to 1.0 for ha-has.

Reply:
@scotje it looks like you were the last one to pin rugged back in https://github.com/puppetlabs/r10k/commit/6c55978d7b5e2c4c57732f3bb4967aa6a31a9a69

I think we probably want to go to a 1.0 release now if you're not aware of any other reasons to block that?

Reply:
Based on the commit message it looks like there was just a specific issue with the `0.24.6` release that we were avoiding. Don't have any more recent context for the 1.x changes.

Reply:
Cool, that's what @mwaggett thought and it what it looked like to me too. Thanks!